code/script.js

```diff
@@ -253,6 +258,11 @@ const setSecret = () => {
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
+  //shows, generates the gameboard
+  generateBoard() 
+  //sets the secret person
+  setSecret()
+  console.log(`Secret person ${secret.name}`)
```

Comment:
If you are sneaky 🐍 you can have a look 🤓 in the inspector and guess right every time  🏆

Reply:
Yes, remember to always clean your code from all console.log()s and commented out code blocks and tips left for you by teachers 🗡️ ✂️ 

BTW you can always do it by typing `secret` to the console 🙈